workshop/content/40-dotnet/50-table-viewer/400-expose-table.md

```diff
@@ -74,7 +74,7 @@ namespace CdkWorkshop
 {
     public class CdkWorkshopStack : Stack
     {
-        public CdkWorkshopStack(Construct scope, string id, IStackProps props = null) : base(scope, id, props)
+        public CdkWorkshopStack(Construct scope, string id) : base(scope, id)
```

Comment:
was this intentional? I thought @RomainMuller wanted this... it's in our init templates

Reply:
Change made before the discussion. Fixed